### ISSUE TITLE
[feature] Customizable screens

### ISSUE THREAD

**Issue by seawolff** [jeremyckahn/farmhand#122]
The ability to change ordering of screens,and maybe to remove the home (1st) screen. Potentially in settings.

Discussion here https://discord.com/channels/714539345050075176/722286738344640552/855538908719874058

**Comment by seawolff** [comment]
Would just also be nice to be able to dismiss the main screen and/or customize it

**Comment by jeremyckahn** [comment]
Yes to this! I posted this idea in the Discord regarding this feature:

> What it the player could just drag-and-drop reorder the screen indicators in #112?
> That'd be a separate PR for sure, but it seems like the most straightforward experience for the player.

**Comment by lstebner** [comment]
being able to hide the home screen was implemented, but do you want to keep this open for the re-ordering @jeremyckahn ? seems like you were interested in that. I'm indifferent here; I don't think I've seen anyone else bring this up so it's up to you.

**Comment by jeremyckahn** [comment]
@lstebner let's close this issue for now as it's quite stale. I'm open to reopening and accepting a Pull Request from anyone who would like to work on it, though!
